src/components/Modal/Modal.jsx

```diff
@@ -15,12 +17,33 @@ export const Modal = ({ setModal, video }) => {
     setModal(false);
```

Comment:
dispatch function should have {type,action}

Reply:
I'm using {type,payload} as arguments while calling the dispatch and {state,action} as parameters in reducers.